<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EndpointOutWidget</class>
 <widget class="QWidget" name="EndpointOutWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>228</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,1">
   <item>
    <widget class="QWidget" name="hexEditPlaceHolder" native="true"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="layoutCheckBoxes">
       <item>
        <widget class="QLabel" name="labelEndpointName">
         <property name="text">
          <string>EP UNKNOWN</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>DUMMY1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_2">
         <property name="text">
          <string>DUMMY2</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="layoutButtons">
       <item>
        <widget class="QPushButton" name="pushButtonWriteOnce">
         <property name="text">
          <string>Write Once</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonKeepWrite">
         <property name="text">
          <string>Keep Write</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
